public/styles/Footer.scss

```diff
@@ -0,0 +1,32 @@
+@import '~bootstrap/scss/functions';
+@import '~bootstrap/scss/variables';
+@import '~bootstrap/scss/mixins';
+
+footer {
+  font-size: 0.8em;
+  bottom: 0;
+  position: absolute;
+  width: 100%;
+  height: 3.75em;
+  line-height: 3.75em; /* Vertically center the text there */
+
+  @media (max-width: 767.98px) {
```

Comment:
Why are we nesting media queries? also can you just make this 768px.

Reply:
@KvNGCzA do you mean adding them inside the footer styling elements?

Reply:
I mean you wrote
`footer {
 // styles
 @media (max-width: 768px) {
  // styles
 }
}
`
But this should be:
`footer {
// styles
}
@media (max-width: 768px) {
// styles
}`
Though what you did works, I am not sure it is best.